pages/blog/2022Q1-summary.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Summary of things that happened around the AsyncAPI Initiative until end of April 2022"
+date: 2022-04-28T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+cover: /img/posts/2022Q1-summary/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Gardner
+excerpt: 'New AsyncAPI brand. AsyncAPI 2.4 and work on 3.0 spec release. Meetings organization changes. Conference organization. AND SO MUCH MORE!'
+featured: true
+---
+
+> TODO here will be link to podcast for those that do not read articles
+
+Welcome to another update for AsyncAPI Initiative. In this article, I want to highlight the most important things around AsyncAPI in the first few months of 2022.
+
+I usually ask the community if folks have anything important that I should put in my updates. Still, it is a pretty much subjective article based mainly on things I see in my personal bubble :smiley:
+
+## Brand Refresh
+
+This one is pretty easy to spot, especially if you have been with AsyncAPI Initiative for some time :smiley:
+
+10th of March, we announced a change in the AsyncAPI brand. Change that was needed. A refresh of this faded green that I personally never liked :joy:
+
+<TwitterTweetEmbed
+  tweetId='1501977719915098120'
+  options={{
+    cards: 'hidden',
+    width: 200,
+    maxWidth: 400
+  }}
+/>
+
+It was a gigantic effort by [Missy Turco](https://twitter.com/missyturco). Months of work, research, and discussions with the community.
+
+It is not all. More work in the area is needed:
+- Design System :muscle: - Get involved in [related repository](https://github.com/asyncapi/design-system) if you are interested in helping out. 
+- AsyncAPI Mascots refresh - We need to refresh our existing Eve and Chan mascots to match the new brand. Any help would be appreciated.
+
+For more regular news and updates on topics around brand and design in AsyncAPI Initiative, I definitely recommend you to follow [Missy Turco](https://twitter.com/missyturco) and read her [regular updates on the subject](https://missyturco.notion.site/03556522990e4067ae877fd236c37268?v=d60154483de849cd8125f8333cfe1d36).
+
+## AsyncAPI Conference
+
+We want to organize the first conference that you not only can watch online but also participate in-person. May is right behind the cornern, not much time left. We need to start working on the event actively now.
+
+We discuss all the aspects of the conference [in Community Discussions](https://github.com/asyncapi/community/discussions/categories/asyncapi-conf-2022-organization) openly. Please join and actively support the effort. We need lots of volunteers to make that happen. 
```

Comment:
```suggestion
We discuss all aspects of our upcoming conference in our [Community Discussion for 2022 Conference Organization](https://github.com/asyncapi/community/discussions/categories/asyncapi-conf-2022-organization) openly. Please join and actively support the effort. We need lots of volunteers to make this happen. 
```
This is another accessibility example where we want to use more specific text in our anchor text.